ksqldb-parser/src/test/java/io/confluent/ksql/parser/tree/CreateConnectorTest.java

```diff
@@ -39,18 +39,18 @@
   public void testEquals() {
     new EqualsTester()
         .addEqualityGroup(
-            new CreateConnector(Optional.of(SOME_LOCATION), NAME, CONFIG, CreateConnector.Type.SOURCE),
-            new CreateConnector(Optional.of(OTHER_LOCATION), NAME, CONFIG, CreateConnector.Type.SOURCE),
-            new CreateConnector(NAME, CONFIG, CreateConnector.Type.SOURCE)
+            new CreateConnector(Optional.of(SOME_LOCATION), NAME, CONFIG, CreateConnector.Type.SOURCE,false),
```

Comment:
nit: code style: need space after `,`:

```suggestion
            new CreateConnector(Optional.of(SOME_LOCATION), NAME, CONFIG, CreateConnector.Type.SOURCE, false),
```

Same for others below.